pkg/yurthub/proxy/remote/remote.go

```diff
@@ -125,3 +129,39 @@ func (rp *RemoteProxy) modifyResponse(resp *http.Response) error {
 	}
 	return nil
 }
+
+func (rp *RemoteProxy) errorHandler(rw http.ResponseWriter, req *http.Request, err error) {
+
+	klog.V(2).Infof("enter remote proxy error handler: %v. error: %v", util.ReqString(req), err)
+	if !rp.cacheMgr.CanCacheFor(req) {
+		klog.Errorf("can not cache for %s", util.ReqString(req))
+		rw.WriteHeader(http.StatusBadGateway)
+		return
```

Comment:
how about write some error message to rw?

Reply:
Refer to the default errHandler, do not write error messages. only return 502